### PR TITLE
[ALLUXIO-2383]add error message

### PR DESCRIPTION
**Pull request by haoxiang47**
https://alluxio.atlassian.net/browse/ALLUXIO-2383

### CODE REVIEW

File: core/server/src/main/java/alluxio/worker/file/FileSystemWorkerClientServiceHandler.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
    * @param worker the file system worker which will handle most of the requests
    */
   public FileSystemWorkerClientServiceHandler(FileSystemWorker worker) {
-    mWorker = Preconditions.checkNotNull(worker);
+    mWorker = Preconditions.checkNotNull(worker,"worker");
```
Comment (HorizonNet) [inline core/server/src/main/java/alluxio/worker/file/FileSystemWorkerClientServiceHandler.java]: Insert a whitespace after the comma.

**Comment by HorizonNet** [comment]
@haoxiang47 I left a comment regarding a missing whitespace. Could you please add the email address used in your commit to your GitHub account (as suggested above). After these changes I will give a LGTM.

**Comment by haoxiang47** [comment]
alluxio-bot, check this please

**Comment by haoxiang47** [comment]
@HorizonNet I pass all the checks, please give some feedback, thank you.

**Comment by HorizonNet** [comment]
LGTM

**Pull request merged**